flows/codelingo/review/review/cli.go

```diff
@@ -166,16 +169,67 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 		return nil, nil, nil, nil, errors.Trace(err)
 
 	}
+	// Build `path` for go projects as "reponame" for a local repo or "github.com/username/reponame" for a github repo
+	// path variable is initially empty and will remain empty for none go repos
+	path := ""
+
+	// Find the current working directory, i.e. where user executed `lingo run review` command on the repo
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, nil, nil, nil, errors.Trace(err)
+
+	}
+	// Check if the repo is a go project
+	goProject, err := isGoProject(wd)
+	if err != nil {
+		return nil, nil, nil, nil, errors.Trace(err)
+	}
+
+	// Check if GOPATH is set
+	goPath := os.Getenv("GOPATH")
+
+	if goPath != "" && goProject {
+		if strings.Contains(wd, "github.com") { // Check if the working directory includes "github.com/username/reponame"
+			swd := wd[strings.Index(wd, "github.com"):]
+			parts := strings.Split(swd, "/")
+			if len(parts) == 3 {
+				path = swd
+			}
+		} else if strings.HasPrefix(wd, goPath+"/src/") { //Check if repo is local and lives in GOPATH/src/reponame
+			swd := wd[strings.Index(wd, "src/"):]
+			parts := strings.Split(swd, "/")
+			if len(parts) == 2 {
+				path = parts[1]
+			}
+		}
+		// If none of the above, path remains empty
+	}
 
 	ctx, cancel := util.UserCancelContext(context.Background())
-	req := &flow.ReviewRequest{
-		Repo:     name,
-		Sha:      sha,
-		Patches:  patches,
-		Vcs:      vcsTypeStr,
-		Dir:      workingDir,
-		Dotlingo: dotlingo,
+
+	var req *flow.ReviewRequest
+	// If path is not empty, it is sent in ReviewRequest and used in go lexicon
+	if path != "" {
```

Comment:
Just return the version with path. Unassigned fields default to the zero value in Go. https://play.golang.org/p/p16l5ISRXBw.

```go
package main

import (
	"fmt"
)

type s struct {
	path string
}

func main() {
	implicit := &s{}
	explicit := &s{
		path: "",
	}
	fmt.Println(implicit.path == explicit.path) // true
}
```